### PR TITLE
pydruid 0.6.5

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Current package info**

| Channel | Downloads | Version | Platforms | License |
| --- | --- | --- | --- | --- |
| defaults | [Conda Downloads](https://anaconda.org/main/pydruid) | [Conda Version](https://anaconda.org/main/pydruid) | [Conda Platforms](https://anaconda.org/main/pydruid) | Conda License |

Changelog: https://github.com/druid-io/pydruid/blob/0.6.5/CHANGELOG.md
License: https://github.com/druid-io/pydruid/blob/0.6.5/LICENSE
Requirements: https://github.com/druid-io/pydruid/blob/0.6.5/setup.py

Actions: 
1. Clean up the recipe
2. Add `--no-build-isolation` to script
3. Remove the requirements/build section
4. Remove pytest-runner from host
5. Update run dependencies: add a comment about keeping optional dependencies at runtime, fix pinning of sqlalchemy <2
6. Add pip check
7. Fix license name
8. Add description

Notes:
- We update this package because of its incompatibility with **sqlalchemy 2.x.x**

### CODE REVIEW

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by varlackc — APPROVED** [review]
Looks Good

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,53 +1,55 @@
...
+    - tornado
 
 test:
   imports:
     - pydruid
     - pydruid.utils
+  requires: 
+    - pip
+  commands: 
+    - pip check
```
Comment (akabanovs) [inline recipe/meta.yaml:47]: Should we add unit tests available?

Comment (skupr-anaconda) [reply recipe/meta.yaml:47]: I've added a test suite

**Review by akabanovs — APPROVED** [review]

**Pull request merged**